raster/src/main/scala/geotrellis/raster/io/geotiff/package.scala

```diff
@@ -24,5 +24,4 @@ package object geotiff extends Implicits {
   val Deflate = DeflateCompression
 
   val GeoTiffReader = reader.GeoTiffReader
-  val TiffTagsReader = reader.TiffTagsReader
```

Comment:
Hm, I thought it was a good pattern to follow, to decompose 'tags' / 'tiffs' parsing from the objects they are collected into. Since parsers are really huge and can be placed into a separate file.